src/package.json

```diff
@@ -13,13 +14,21 @@
     "eslint-import-resolver-typescript": "^2.7.1",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-prettier": "^4.0.0",
```

Comment:
Huh.. I guess I just followed the example code from @nemzes on Slack. With that said, it's not surprising the prettier team is preaching "don't wrap me". :wink: I guess we could run prettier first and eslint second, but I fear it will complicate things if we require them to run in a specific order instead of just running one tool where we can configure the order. I checked now, and while I haven't tested it to make sure, it seems if you configure WebStorm to run both eslint and prettier on save, it runs eslint first and prettier second - which would conflict with having something else happen automatically on commit.

Or, I may just have assumed we'll want to run both eslint and prettier on commit - but maybe we just want one of them? I believe we can use [import-js/eslint-plugin-import](https://github.com/import-js/eslint-plugin-import) to automatically use `import type` and group these type imports separately. I've seen that being a common feedback in code review, so it would be nice to just automate it for us. For that we'd want to run both tools on commit.

Reply:
Not sure what is best :) I just thought since its not recommended, maybe its good to follow that. But if it complicates other things, it may not be worth to follow those recommendations.

+1 for using [import-js/eslint-plugin-import](https://github.com/import-js/eslint-plugin-import) - I didnt know about that one. Seems like a useful addition.

Reply:
Lets leave it in for now, and revisit if it actually causes issues :)

Reply:
Agree! We'll quickly notice if it's too slow or buggy.